packages/types/src/index.d.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-unassigned-import
+import './global';
```

Comment:
Do we want to always declare the global? This package is used elsewhere in the codebase

Reply:
It's not needed elsewhere, but this seems like the easiest way to do it, so you can import it as `node_modules/@metamask/snap-types` in your `tsconfig.json`.

Reply:
Yeah, I just don't know if we want to "polute" the global in other contexts. If it wasn't imported here could you import it in tsconfig as `node_modules/@metamask/snap-types/global.d.ts`?

Reply:
That would be possible, if I move it to the root folder

Reply:
I agree that it would be better to keep the global declaration separate from all other types. You should be able to import any individual type without modifying the global.